default-config.xml.dist

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<xulieta xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:noNamespaceSchemaLocation="./vendor/codelicia/xulieta/xulieta.xsd">
+    <exclude>vendor</exclude>
+    <exclude>node_modules</exclude>
+</xulieta>
```

Comment:
Maybe this file should have the default validator for PHP and Parser for Markdown.

Reply:
Linking #92